models/addresses/base/addresses_base_dex.sql

```diff
@@ -0,0 +1,29 @@
+{{ config(
+    schema = 'addresses_base',
```

Comment:
i communicated that poorly, but i meant to add `schema` config here (i pushed a commit)

that actually worked out though, cause adding schema yml file was good too 🙂